client/src/components/Toolbar/Toolbar.scss

```diff
@@ -14,6 +14,41 @@
   .btn {
     margin-bottom: 0;
   }
+
+  .right {
+    margin-left: auto;
+
+    .btn-group:not(last-child) {
+      margin-right: 10px !important;
+    }
+  }
+
+  .rounded {
+
+    &.btn,
+    >.btn {
+      width: 32px;
+      padding-left: 7px;
```

Comment:
There's some specific styling going on here. It probably makes sense to define these "betterbuttons" styles in BEM format? Perhaps there's some existing variables could be used for the padding here. There's some like `btn-spacing` and `panel-padding`.

Reply:
This could also allow adding a background colour on the previous and next buttons that the designs had and keep it specific to these "betterbuttons"